apis/v1alpha2/httproute_types.go

```diff
@@ -657,23 +657,31 @@ type HTTPRequestRedirect struct {
 	// header in the response.
 	// When empty, the protocol of the request is used.
 	//
+	// Support: Extended
```

Comment:
For GCP we have some cases where we just support same protocol or HTTP->HTTPS, not HTTPS->HTTP. I think that means this field should be extended.